src/LegendList.tsx

```diff
@@ -1186,6 +1209,7 @@ const LegendListInner: <T>(props: LegendListProps<T> & { ref?: ForwardedRef<Lege
                 ListEmptyComponent={data.length === 0 ? ListEmptyComponent : undefined}
                 maintainVisibleContentPosition={maintainVisibleContentPosition}
                 scrollEventThrottle={scrollEventThrottle ?? (Platform.OS === "web" ? 16 : undefined)}
+                waitForInitialLayout={true}
```

Comment:
@jmeistrich I think that waitForInitialLayout should be default option, and should be opted out when needed. WDYT?

Reply:
But this would always set it to true - should initialize the value to default to true at the top of the component?

Reply:
Yeah, you're right, i need to set default prop. Let me fix that!